topics/reactphp/index.md

```diff
@@ -0,0 +1,11 @@
+---
+related: php
+created_by: Igor Wiedler
+display_name: ReactPHP
+github_url: https://github.com/reactphp
+released: April 14, 2012
+short_description: Event-driven, non-blocking I/O with PHP.
```

Comment:
Could you expand this a bit? Maybe "ReactPHP is a library that allows event-driven, non-blocking I/O with PHP."